.golangci.yml

```diff
@@ -0,0 +1,23 @@
+version: "2"
+run:
+  timeout: 240m
+  go: "1.24"
+  modules-download-mode: readonly
+  allow-parallel-runners: true
+output:
+  formats:
+    text:
+      path: stdout
+    junit-xml:
+      path: report.xml
+linters:
+  default: all
+  disable:
+    - wsl
+    - depguard
+    - exhaustruct
```

Comment:
this might be useful

Reply:
Maybe with some reasonable settings. Otherwise, it's overwhelming.